src/pages/networks/forms/NetworkFormMain.tsx

```diff
@@ -61,91 +61,93 @@ const NetworkFormMain: FC<Props> = ({ formik, project }) => {
           )}
         </Col>
       </Row>
-      <ConfigurationTable
-        rows={[
-          getConfigurationRow({
-            formik,
-            name: "ipv4_address",
-            label: "IPv4 address",
-            defaultValue: "auto",
-            children: (
-              <IpAddressSelector
-                id="ipv4_address"
-                address={formik.values.ipv4_address}
-                setAddress={(value) => {
-                  void formik.setFieldValue("ipv4_address", value);
+      {formik.values.networkType !== "physical" && (
```

Comment:
for my understanding, so a managed network can be of type `physical`? How do we create that kind of network?

Reply:
Yes, see the linked issue. The user was using microcloud setup that creates a managed physical network.